controller/controller.go

```diff
@@ -45,17 +46,20 @@ func (c *Controller) reconcile() error {
 		return fmt.Errorf("listing resources: %w", err)
 	}
 
+	var merr error
 	if err = c.Resources.ReconcileNamespaces(resources); err != nil {
-		return fmt.Errorf("reconciling namespaces: %w", err)
+		c.Log.Error("error reconciling namespaces", "err", err)
```

Comment:
I'm wondering if the log messages here are necessary since there is also a log message in `ctrl.reconcile()` :thinking: ?.. Any errors here will get logged twice.

Reply:
I guess I was thinking I should log the error immediately, but now that I'm seeing this again, you're right. I'll remove these log lines!